.packit.yaml

```diff
@@ -0,0 +1,12 @@
+specfile_path: edd.spec
+
+actions:
+  create-archive:
+  - make tarball
```

Comment:
Works for me locally on the packit `master`. Let's wait for the redeploy.

Reply:
Nice! Looking forward :)